modules/gapi/test/render/gapi_render_tests_ocv.cpp

```diff
@@ -97,6 +145,19 @@ TEST_P(RenderNV12OCVTestFTexts, AccuracyTest)
                                 })));
 }
 
+TEST_P(RenderFrameNV12OCVTestFTexts, AccuracyTest)
+{
+    // G-API code //////////////////////////////////////////////////////////////
+    cv::gapi::wip::draw::Prims prims;
+    prims.emplace_back(cv::gapi::wip::draw::FText{ text, org, fh, color });
+    cv::MediaFrame nv12 = cv::MediaFrame::Create<TestMediaNV12>(y_gapi_mat, uv_gapi_mat);
+    EXPECT_NO_THROW(cv::gapi::wip::draw::render(nv12, prims,
+        cv::compile_args(cv::gapi::wip::draw::freetype_font{
+        "/usr/share/fonts/truetype/wqy/wqy-microhei.ttc"
+            })));
```

Comment:
Maybe it's worth to add some check that images were changed

Reply:
Done